components/ui/button.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const Button = ({ click, text, size, variant }) => {
+
+    let buttonStyles = '';
+    function buttonType() {
```

Comment:
These function can also be improved, but we'll discuss this one in the office, because it's a more complex thing to implement, and it requires more packages to install